app/models/link_report.rb

```diff
@@ -29,6 +29,7 @@ def generate_links(parsed_content)
     parsed_content.each do |content|
       links << list_contents(content.fetch(:contents)) unless content.fetch(:type, nil) != 'list'
     end
-    links.flatten!
+    links.flatten! unless links.empty?
+    links
```

Comment:
Is this change related to adding `request_broken_links`? If not, can you squash it up to the previous commit?